internal/integration-tests/equipment/integration_test.go

```diff
@@ -512,18 +512,179 @@ func TestIntegration_DeleteEquipment(t *testing.T) {
+		require.Error(t, err)
+
+		wantErr := equipment.NewBlockEquipmentDefault(http.StatusForbidden)
+		wantErr.Payload = &models.Error{Data: &models.ErrorData{
+			Message: "You don't have rights to block the equipment",
+		}}
+		assert.Equal(t, wantErr, err)
+	})
+
+	t.Run("Block Equipment is permitted for managers", func(t *testing.T) {
+		tokens := utils.ManagerUserLogin(t)
+		auth := utils.AuthInfoFunc(tokens.GetPayload().AccessToken)
+		model, err := setParameters(ctx, client, auth)
+		require.NoError(t, err)
+		eq, err := createEquipment(ctx, client, auth, model)
+		require.NoError(t, err)
+
+		params := equipment.NewBlockEquipmentParamsWithContext(ctx).WithEquipmentID(*eq.Payload.ID)
+		params.Data = &models.ChangeEquipmentStatusToBlockedRequest{
+			StartDate: strfmt.DateTime(startDate),
+			EndDate:   strfmt.DateTime(endDate),
+		}
+
+		res, err := client.Equipment.BlockEquipment(params, auth)
+		require.NoError(t, err)
+		require.True(t, res.IsCode(http.StatusNoContent))
+	})
+
+	t.Run("Block Equipment with active orders", func(t *testing.T) {
+		tokens := utils.ManagerUserLogin(t)
+		auth := utils.AuthInfoFunc(tokens.GetPayload().AccessToken)
+		model, err := setParameters(ctx, client, auth)
+		require.NoError(t, err)
+		eq, err := createEquipment(ctx, client, auth, model)
+		require.NoError(t, err)
+
+		orStartDate, orEndDate := time.Now(), time.Now().AddDate(0, 0, 1)
+		firstOrderID, err := createOrder(ctx, client, auth, eq.Payload.ID, orStartDate, orEndDate)
+		require.NoError(t, err)
+		orStartDate, orEndDate = time.Now().AddDate(0, 0, 2), time.Now().AddDate(0, 0, 3)
+		secondOrderID, err := createOrder(ctx, client, auth, eq.Payload.ID, orStartDate, orEndDate)
+		require.NoError(t, err)
+
+		listParams := orders.NewGetAllOrdersParamsWithContext(ctx)
+		ordersList, err := client.Orders.GetAllOrders(listParams, auth)
+		require.NoError(t, err)
+		for i, o := range ordersList.Payload.Items {
+			var st string
+			if i == 0 {
+				st = domain.OrderStatusApproved
+			} else {
+				st = domain.OrderStatusRejected
+			}
+
+			dt := strfmt.DateTime(time.Now())
+			osp := orders.NewAddNewOrderStatusParamsWithContext(ctx)
+			osp.Data = &models.NewOrderStatus{
+				OrderID:   o.ID,
+				CreatedAt: &dt,
+				Status:    &st,
+				Comment:   &st,
+			}
+			_, err = client.Orders.AddNewOrderStatus(osp, auth)
+			require.NoError(t, err)
+		}
+
+		params := equipment.NewBlockEquipmentParamsWithContext(ctx).WithEquipmentID(*eq.Payload.ID)
+		params.Data = &models.ChangeEquipmentStatusToBlockedRequest{
+			StartDate: strfmt.DateTime(startDate),
+			EndDate:   strfmt.DateTime(endDate),
+		}
+
+		res, err := client.Equipment.BlockEquipment(params, auth)
+		require.NoError(t, err)
+		require.True(t, res.IsCode(http.StatusNoContent))
+
+		orders, err := client.Orders.GetOrdersByStatus(
+			orders.NewGetOrdersByStatusParamsWithContext(ctx).WithStatus(domain.OrderStatusBlocked), auth)
+		fmt.Println(orders)
```

Comment:
Please replace by `require.NoError(t, err)` + some chek for orders (len or not-nil, idk)